docs/user/reference/environment-variables.rst

```diff
@@ -117,6 +117,11 @@ All :doc:`build processes </builds>` have the following environment variables au
     :Example: ``feature/signup``
     :Example: ``update-readme``
 
+    .. note::
+
+       When building PRs, this variable contains the commit hash of the PR branch,
+       as we don't have access to the branch name.
```

Comment:
Wonder if we should return the PR number instead, since it's more on the side of being a name.

Reply:
Why don't we have access to the branch name?

Reply:
We just have access to the PR number, we don't save the name of the source branch, and even if we do so, we would need to decide how to display the source branch in case of forks.

Reply:
If `READTHEDOCS_GIT_COMMIT_HASH` already contains the commit hash, I think the number makes sense.